server/tmserver/world.py

```diff
@@ -117,6 +119,40 @@ def handle_look(cls, sender_obj, action_args):
         for o in cls.area_of_effect(sender_obj):
             o.handle_action(cls, sender_obj, 'look', action_args)
 
+    @classmethod
+    def handle_refresh(cls, sender_obj, action_args):
+        # TODO: should the server dictate what text goes into the client's info
```

Comment:
Definitely the latter. we don't want the server to ever have to think about the UI -- its only job is passing data (which sometimes might be as simple as a string of text to show or as complex as a big json pile)